NextJs/pages/api/hello.js

```diff
@@ -1,5 +0,0 @@
-// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
-
-export default function handler(req, res) {
```

Comment:
@ali-cash is this file being deleted then?

Reply:
yes its completely unnecesary